src/files.ts

```diff
@@ -220,10 +249,14 @@ async function* searchFiles(
   patterns: string[],
   cwd: string,
   symlinksEnabled: boolean,
-  ignores: string[],
+  policies?: FilePolicies,
```

Comment:
q: Why is this optional? Shouldn't it always be passed?

Reply:
Wasn't sure whether it should be or not, it's not really needed for this to work, but might be cleaner to have it required than doing null checks/coalescing. Will change.